STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.source.boundary.JavaSourceContainerTest.testAddJavaSourceDoublette
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/source/boundary/JavaSourceContainerTest.java
CLEANER: de.strullerbaumann.visualee.source.boundary.TestGroup100Case0.test244
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/source/boundary/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 20
NEW CLEANER SIZE: 2
ELAPSED TIME: 65.48200011253357
==========================
@@ -63,0 +63,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
